sdk/go/common/apitype/updates.go

```diff
@@ -95,6 +96,9 @@ type StartUpdateResponse struct {
 
 	// Token is the lease token (if any) to be used to authorize operations on this update.
 	Token string `json:"token,omitempty"`
+
+	// TokenExpiration is the point in time by which the token will expire.
+	TokenExpiration *time.Time `json:"tokenExpiration,omitempty"`
```

Comment:
Everywhere else we send times from the service we send them as unix timestamps (int64). This would be different as go formats time.Time like " 2014-05-16T08:28:06.801064-04:00" 
Is that intended?

Reply:
I just was not paying attention. There seem to be both versions in use? In addition there's int64+*int64 in use.

```go
// ManifestV1 captures meta-information about this checkpoint file, such as versions of binaries, etc.
type ManifestV1 struct {
	// Time of the update.
	Time time.Time `json:"time" yaml:"time"`

// OperationStatus describes the state of an operation being performed on a Pulumi stack.
type OperationStatus struct {
	Kind    UpdateKind `json:"kind"`
	Author  string     `json:"author"`
	Started int64      `json:"started"`
}
// LogEntry is the individual entries in a JSON response to a Logs operation.
type LogEntry struct {
	ID        string `json:"id"`
	Timestamp int64  `json:"timestamp"`
	Message   string `json:"message"`
}
// StackSummary describes the state of a stack, without including its specific resources, etc.
type StackSummary struct {
	// LastUpdate is a Unix timestamp of the start time of the stack's last update, as applicable.
	LastUpdate *int64 `json:"lastUpdate,omitempty"`
```

I'm fine with using `*in64` or whatever makes sense.

Reply:
@isabeldepapel in case service team has a preference here.

Reply:
No, no preference--whatever you all thinks makes the most sense works for us

Reply:
I think int64 would be more consistent. The service doesn't actually write ManifestV1's.